src/Auth/Auth.styles.ts

```diff
@@ -0,0 +1,243 @@
+import styled, { css } from "styled-components";
+
+import { cssScale, defaultFontSize } from "src/common/constants/cssScale";
+import { colors } from "src/common/constants/colors";
+
+interface AuthContainerProp {
+  signUpPanelActive: boolean;
+}
+
+const { c0, c3, c4, c5, c7, c10, c11 } = cssScale;
+
+export const AuthenticationWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: ${colors.background};
+  height: 100vh;
+  align-items: center;
+  margin-top: -${c5};
+
+  p {
+    font-size: ${defaultFontSize};
+    line-height: ${c5};
+    letter-spacing: 0.5px;
```

Comment:
Do you want to move these to `GlobalStyles`?

Reply:
no this was button-specific for now. we can modify in the future if we have requirements like that but not needed as of now.

Reply:
Let's use class selectors for these then